games-action/minetest_game/Manifest

```diff
@@ -1,2 +1,7 @@
 DIST minetest_game-0.4.13.tar.gz 1371339 SHA256 13a8b474b439da326f627290b06065f7f5adff12d8462f29114ba818653ec604 SHA512 41f48e0abda47fb45750ff21128fb8249b5f2da9bd63dfa52ce438fe482725190a1161faf76c43664e38916f61615be4b9b8914efd88cd02d6ba498b2f69dd62 WHIRLPOOL 02836d6d9a2fe9b66f49983ac2caf88825808b340033cdecfcf84b2ebaee76e772604fd9537403059208fb7bf0ced6f3533356b87e0868a73a64f6d5436cacd8
 DIST minetest_game-0.4.14.tar.gz 772754 SHA256 0547524c75f8649002e2280ba2e623c1af4a1abf2f5ea3c5128feaecc3e88962 SHA512 c89e9e4bc55765d36ad84c0ab30ba4a75edff7bb57f1a2890c7fd2f8ca4f1bb9d12bb6e4d5a633913cacecf510683f3b062a0a166a2856f71cc9cda91e048eaf WHIRLPOOL 6b2fd4d14a8939e963bd0d8dde54c6f9d8de9d9ca00d98e93cf1de48950a820ad95f97574119ab1269210b8709f2fcc2cf6602f4cd2c740867af9246481b9090
+DIST minetest_game-0.4.15.tar.gz 1345800 SHA256 db787664f1c1d42d2189ba6df4ea9b8b3e0d61c54da66009ad88b2075e98bfd5 SHA512 708a9be15308850fa9545f2ec727bc61b32f1f7401ba70df7128fc91c715ce4c066802ca4a37dd1abe0f921bc61fdae27c91cef08885721c66e8147c1e3f026c WHIRLPOOL 34e03fc976948afefda81390b4659081a675f6756f5186a81af52a81536856692897058a4d49d4e1764ae6bdc9722f4f3cb7a56c2bb3a7733ae3ed0d2566ef3e
+EBUILD minetest_game-0.4.13.ebuild 589 SHA256 7b52e0d19928a05b94c9924037441c93d8860d7711766503e09c78b380bee013 SHA512 666d33e55042c9253402cefc8ad4b286c33b97292e7620edf436445533abc9ad61550b6ae8252942ced3360c340d19f3304ad5072c93d1165fed3f0ad6169214 WHIRLPOOL bd26e1dc9510ee9ea99f4bbefa0fd49a3aec292278003cd2b435b2c9d8b89943a55332b3f24d8f2541130f09611acc206c29e745a16d7ba1878c51e53b9b5d6b
+EBUILD minetest_game-0.4.14.ebuild 581 SHA256 4281c64c1a0b0f17534fd534fd72af8722406a0190a50107b6ba0304a2ec1dfa SHA512 8e953175f5dfd5e99de7223ba754b63d6259528681ffdea018118002f29b9e22c5508598df168ea74a7fe618ffc4e0cb2bb2bf8655169ce97775f0a5cc7607ac WHIRLPOOL 8c13e645121b5f95e8075ecd80300fe190da42b16afe14ebafa4aa7962bc5239afdc84b7ec79258111e880401f6cc9f36d818ccf2f16cc74ea422161762c9418
+EBUILD minetest_game-0.4.15.ebuild 581 SHA256 4281c64c1a0b0f17534fd534fd72af8722406a0190a50107b6ba0304a2ec1dfa SHA512 8e953175f5dfd5e99de7223ba754b63d6259528681ffdea018118002f29b9e22c5508598df168ea74a7fe618ffc4e0cb2bb2bf8655169ce97775f0a5cc7607ac WHIRLPOOL 8c13e645121b5f95e8075ecd80300fe190da42b16afe14ebafa4aa7962bc5239afdc84b7ec79258111e880401f6cc9f36d818ccf2f16cc74ea422161762c9418
```

Comment:
this is a think manifest. You can either use

1. `repoman` + GPG (recommended, takes a bit of effort to setup)
2. `ebuild` manifest

to get rid of the EBUILD lines. The CI is failing due to these lines